src/open_inwoner/react/i18n/locales/nl.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Q6fIiw": {
+    "defaultMessage": "This is the current count",
+    "description": "Counter",
+    "originalDefault": "This is the current count"
```

Comment:
That is not Dutch :-)

Also when I run the project and build etc. I get some extra files, and not sure if we should add these to Gitignore?:

![Screenshot 2025-07-07 at 09 51 31](https://github.com/user-attachments/assets/70e28dd5-f0bc-421f-a771-f81a292640dd)